src/Resources/themes/pocketalice/Default/base.html.twig

```diff
@@ -4,4 +4,4 @@
   Pocket Alice Website
 {% endblock %}
 
-{% set google_play_url = "https://play.google.com/store/apps/details?id=org.catrobat.catroid" %}
+{% set google_play_url = "https://catrob.at/gp" %}
```

Comment:
if all have the same URL. remove the variable overwrites from the themes and just set a default value.